app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: int, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.spouse = None
+        Person.people[name] = self
+
+    def set_spouse(self, spouse_name: int) -> None:
+        if spouse_name in Person.people:
+            self.spouse = Person[spouse_name]
```

Comment:
You will need to set `husband` or `wife` attributes to `Person` instances, not their names.